Modules/socketmodule.c

```diff
@@ -2127,6 +2159,14 @@ getsockaddrlen(PySocketSockObject *s, socklen_t *len_ret)
     }
 #endif
 
+#if defined(AF_QIPCRTR)
+    case AF_QIPCRTR:
+    {
+        *len_ret = sizeof (struct sockaddr_qrtr);
```

Comment:
please remove the extra space after sizeof

Reply:
For the record, I'm ignoring this to follow the coding style in this function.